modules/nested-tabs-html/assets/js/frontend/handlers/nested-tabs-html.js

```diff
@@ -321,8 +325,12 @@ export default class NestedTabsHtml extends Base {
 		}
 	}
 
-	changeActiveTabByKeyboard( event, tabIndex ) {
-		this.changeActiveTab( tabIndex, true );
+	changeActiveTabByKeyboard( event, settings ) {
+		if ( settings.widgetId !== this.getID() ) {
```

Comment:
Verify that this is triggered by the correct widget.

Reply:
I am using this 'trigger' event in here assets/dev/js/frontend/handlers/accessibility/nested-title-keyboard-handler.js: 

elementorFrontend.elements.$window.trigger( 'elementor/nested-elements/activate-by-keyboard', { widgetId: this.getID(), titleIndex } );

This triggers every function that is listening to this trigger and can trigger an incorrect Tabs or Menu widget.